pkg/proc/native/proc_ebpf_linux.go

```diff
@@ -1,6 +1,3 @@
-//go:build ebpf
-// +build ebpf
-
 package native
```

Comment:
shouldn't this have the same build tags as the rest of the ebpf stuff (linux && amd64 && go1.17 && cgo)?

Reply:
Yeah, you're right.